rasa/train.py

```diff
@@ -127,10 +134,12 @@ def train(
         train_path: Directory in which to train the model.
         output_path: Output path.
         force_training: If `True` retrain model even if data has not changed.
+        fixed_model_name: Name of model to be stored.
```

Comment:
You changed the order of arguments, can you please change it back to its original order as this is a public method? Thanks.